src/client/cmd/connect.cpp

```diff
@@ -52,35 +52,40 @@ mp::ReturnCode cmd::Connect::run(mp::ArgParser* parser)
         }
         catch (const std::exception& e)
         {
-            cerr << "connect failed: " << e.what() << "\n";
+            cerr << "shell failed: " << e.what() << "\n";
             return ReturnCode::CommandFail;
         }
         return ReturnCode::Ok;
     };
 
     auto on_failure = [this](grpc::Status& status) {
-        cerr << "connect failed: " << status.error_message() << "\n";
+        cerr << "shell failed: " << status.error_message() << "\n";
         return ReturnCode::CommandFail;
     };
 
     return dispatch(&RpcMethod::ssh_info, request, on_success, on_failure);
 }
 
-std::string cmd::Connect::name() const { return "connect"; }
+std::string cmd::Connect::name() const { return "shell"; }
+
+std::vector<std::string> cmd::Connect::aliases() const
+{
+    return {name(), "sh", "connect"};
+}
```

Comment:
Strange that Connect's name is "shell" and that "connect" is only an alias. I know it's avoiding code churn, but I'd prefer we stick to naming classes after their primary command name.